### ISSUE TITLE
Running kubric with singularity instead of docker

### ISSUE THREAD

**Issue by pratogab** [google-research/kubric#230]
Hi,

I am just curious if anyone has successfully managed to run kubric using singularity instead of docker and if so how? Trying to do so on my side, but encountering some issues.

**Comment by MrXandbadas** [comment]
Yes, it is tough, but I have managed to install it locally although it needed alot of hoop-jumping and I went though hundreds of tabs...

This is why its set up with docker.

If you want to build it locally you will have to build (not only) the blender python api extension which can be a real hassle (on its own) but also make sure all the dev requirements are installed and accessible across the whole codebase.
